frontend/src/utils/fabric.utils.ts

```diff
@@ -134,3 +135,14 @@ export const deleteObject = (canvas: fabric.Canvas) => {
 		document.removeEventListener('keydown', objectDeleteHandler);
 	});
 };
+
+export const setObjectIndexLeveling = (canvas: fabric.Canvas) => {
+	canvas.on('object:added', (e) => {
+		const postits = canvas._objects.filter((obj) => obj.type === 'postit');
```

Comment:
```suggestion
		const postits = canvas._objects.filter((obj) => obj.type === CanvasType.postit);
```

타입 들고와서 비교하는게 더 정확할 것 같습니다!

Reply:
이건 성훈님꺼랑 합치고 나서 다시 설정해보겠습니다!